lib/routers/profile/person.js

```diff
@@ -96,10 +96,24 @@ const getSingleProfile = req => {
 
 const router = Router({ mergeParams: true });
 
+function mapSpeciesFromModules(cert) {
+  if (cert.species && cert.species.length) {
+    return cert;
+  }
+  return {
+    ...cert,
+    species: (cert.modules || [])
+      .reduce((arr, module) => [ ...arr, ...(module.species || []) ], [])
```

Comment:
Minor quibble, but avoid using `module` as a variable name as it's already a keyword.

Reply:
ah that's a good point - I'll update